<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Particulas</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="0">
        <widget class="QPushButton" name="insertar_inicio">
         <property name="text">
          <string>Insertar al Inicio</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="ox">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <widget class="QPushButton" name="OrdenarAscendenteVelocidad_pushbutton_2">
         <property name="text">
          <string>Ordenar de forma Ascendente Velocidad</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Velocidad ( KM/h )</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Origen X ( 0-500 )</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="green">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QPushButton" name="OrdenarAscendenteID_pushbutton_2">
         <property name="text">
          <string>Ordenar de forma Ascendente ID</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="insertar_final">
         <property name="text">
          <string>Insertar al Final</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="velocidad">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="blue">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="red">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Destino Y  ( 0-500 )</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Destino X  ( 0-500 )</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="oy">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="dy">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QPushButton" name="OrdenarDescendenteDistancia_pushbutton_2">
         <property name="text">
          <string>Ordenar de forma Descendente Distancia</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>BLUE ( 0-255 )</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Origen Y ( 0-500 )</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>GREEN ( 0-255 )</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="dx">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>RED ( 0-255 )</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QPushButton" name="randomPushButton">
         <property name="text">
          <string>Generar Particula Aleatoria</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Particulas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="lista_particulas"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0" colspan="2">
         <widget class="QLineEdit" name="buscar_lineedit">
          <property name="placeholderText">
           <string>Escribe un ID a Buscar...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="Buscar_pushbutton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tabla"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gráfico</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="2">
         <widget class="QPushButton" name="mostrarPuntosMasCercanosPushButton_3">
          <property name="text">
           <string>Mostrar Puntos más Cercanos</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="mostrarPuntosPushButton_2">
          <property name="text">
           <string>Mostrar Solo Puntos</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="mostrarGraficoPushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="accionAbrir"/>
    <addaction name="accionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="accionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="accionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
